src/medida/stats/snapshot.cc

```diff
@@ -22,24 +22,55 @@ static const double kP999_Q = 0.999;
 
 class Snapshot::Impl {
  public:
-  Impl(const std::vector<double>& values);
-  ~Impl();
-  std::size_t size() const;
-  double getValue(double quantile) const;
-  double getMedian() const;
-  double get75thPercentile() const;
```

Comment:
We chatted offline, but for future reference: I remove `const` since `CKMSSample` has a member `CKMS` which updates its state when calculating the quantile. But I learned that I can just have a pointer to `CKMS`, so I reverted the change (now they are back to being `const`).

Reply:
Thanks.